source/Tipster.cpp

```diff
@@ -62,15 +62,21 @@ Tipster::Tipster()
 
 	fTipsterTextView = new TipsterText();
 	fIcon = new BButton("iconview", "", new BMessage(OPEN_URL));
+	fIcon -> SetToolTip( B_TRANSLATE("Open the Tipster homepage") );
 
 	BDragger* dragger = new BDragger(this);
 
 	BLayoutBuilder::Group<>(this, B_HORIZONTAL, 0)
-			.SetInsets(-2, -2, 0, -2)
-			.Add(fIcon)
+			.SetInsets(1, 1, 1, 1)
```

Comment:
Those insets were on purpose to avoid the smal spacing around the icon and have it flush with the view borders. It should be kept as is, IMO.

Reply:
Maybe that's a question to discuss about :-) For me this way looks more as a button as when it's completely attached